File: Mileage_Matters.java
import java.util.*;
import java.lang.*;
import java.io.*;

class Mileage_Matters
{
	public static void main (String[] args) throws java.lang.Exception
	{
		Scanner scan = new Scanner (System.in);
		int t = scan.nextInt();
		while(t-->0){
		    int n = scan.nextInt();
		    int x = scan.nextInt();
		    int y = scan.nextInt();
		    int a = scan.nextInt();
		    int b = scan.nextInt();
		    double p = ((float)n/a)*x;
		    double d = ((float)n/b)*y;
		    if (p<d)
		    System.out.println("PETROL");
		    else if (d<p)
		    System.out.println("DIESEL");
		    else
		    System.out.println("ANY");
		}

	}
}

File: MileageMatters.kt
fun main(args:Array<String>){
    repeat(readLine()!!.toInt()){
        var(n,x,y,a,b) = readLine()!!.split(" ").map(String::toInt)
        var petrol = (n+0.0)/a*x
        var diesel = (n+0.0)/b*y
        if(petrol<diesel)
            println("PETROL")
        else if (petrol==diesel)
            println("ANY")
        else
            println("DIESEL")
    }
}
